miio/integrations/vacuum/roborock/vacuum.py

```diff
@@ -958,6 +959,52 @@ def set_child_lock(self, lock: bool) -> bool:
         """Set child lock setting."""
         return self.send("set_child_lock_status", {"lock_status": int(lock)})[0] == "ok"
 
+    def _verify_mop_dryer_supported(self) -> None:
+        """Checks if model supports mop dryer add-on."""
+        # dryer add-on is only supported by following models
+        if self.model not in [ROCKROBO_S7, ROCKROBO_S7_MAXV]:
```

Comment:
Btw, could you run `get_fw_features` raw command and report the information you get at https://github.com/marcelrv/XiaomiRobotVacuumProtocol/blob/master/fw_features.md ? Maybe we can remove the hardcoded models and use the device-provided information in the future :-)

Reply:
Will do :)